pyshelf/endpoint_decorators.py

```diff
@@ -153,4 +154,25 @@ def wrapper(container, *args, **kwargs):
 
         return wrapper
 
+    def decode_request(self, func):
+        """
+            Will decode and inject the data if it is valid json
+            otherwise a error will be returned.
```

Comment:
You are right.  Same with the error message below.  Don't know what to call it (or what to say).  Ideas?